python/paddle/distribution/multinomial.py

```diff
@@ -142,7 +147,7 @@ def log_prob(self, value):
             + (value * logits).sum(-1)
         )
 
-    def sample(self, shape=()):
+    def sample(self, shape: tuple[int]) -> Tensor:
```

Comment:
```suggestion
    def sample(self, shape: Iterable[int]) -> Tensor:
```

用 `Iterable`  吧，下面代码用的这个做判断 ～